app/models/hmcts_common_platform/offence.rb

```diff
@@ -40,24 +42,8 @@ def allocation_decision_mot_reason_code
       data.dig(:allocationDecision, :motReasonCode)
     end
 
-    def laa_appln_reference_status_code
-      data.dig(:laaApplnReference, :statusCode)
-    end
-
-    def laa_appln_reference_status_date
-      data.dig(:laaApplnReference, :statusDate)
-    end
-
-    def laa_appln_reference_end_date
-      data.dig(:laaApplnReference, :effectiveEndDate)
-    end
-
-    def laa_appln_reference_status_description
-      data.dig(:laaApplnReference, :statusDescription)
-    end
-
-    def laa_appln_reference_laa_contract_number
-      data.dig(:laaApplnReference, :laaContractNumber)
+    def laa_reference
```

Comment:
Thinking this can be private. In the same way `person_defendant` is private in `Defendant`